pydis_site/templates/home/timeline.html

```diff
@@ -12,6 +12,333 @@
+          </div>
+
+          <div class="flex justify-between items-center">
+            <span class="cd-timeline__date">July 21st, 2022</span>
+          </div>
+        </div>
+      </div>
+
+      <div class="cd-timeline__block">
+        <div style="background-color: grey;" class="cd-timeline__img cd-timeline__img--picture">
+          <i class="fa fa-handshake"></i>
+        </div>
+
+        <div class="cd-timeline__content text-component">
+          <h2>Partnership with pyqtgraph</h2>
+          <p class="color-contrast-medium">
+            The <code>#pyqtgraph</code> channel is created for the Scientific Graphics and GUI
+            library pyqtgraph, joining <code>#black-formatter</code>.
+          </p>
+          <div class="flex justify-between items-center">
+            <span class="cd-timeline__date">May 19th, 2022</span>
+          </div>
+        </div>
+      </div>
+
+      <div class="cd-timeline__block">
+        <div class="cd-timeline__img pastel-blue cd-timeline__img--picture">
+          <i class="fa fa-robot"></i>
+        </div>
+
+        <div class="cd-timeline__content text-component">
+          <h2>Addition of @Sir Robin</h2>
+          <p class="color-contrast-medium">
+            Our arsenal of bots grows! We add @Sir Robin to power and manage all of our future
+            events and to support the Events Team.
+          </p>
+          <div class="flex justify-between items-center">
+            <span class="cd-timeline__date">Feb 21st, 2022</span>
+          </div>
+        </div>
+      </div>
+
+      <div class="cd-timeline__block">
+        <div class="cd-timeline__img pastel-green cd-timeline__img--picture">
+          <i class="fa fa-question"></i>
+        </div>
+
+        <div class="cd-timeline__content text-component">
+          <h2>Trivia Night</h2>
+          <p class="color-contrast-medium">
+            How well do you know Python inside out? Members got to find out in a Trivia Night event.
+            Contestants were given questions about Python's internals, its development, and more.
+            To win, contestants had to get the most questions right while being fast to answer.
+          </p>
+          <div class="flex justify-between items-center">
+            <span class="cd-timeline__date">Feb 12th, 2022</span>
+          </div>
+        </div>
+      </div>
+
+      <div class="cd-timeline__block">
+        <div class="cd-timeline__img cd-timeline__img--picture">
+          <img src="{% static "images/timeline/cd-icon-picture.svg" %}" alt="Picture">
+        </div>
+
+        <div class="cd-timeline__content text-component">
+          <h2>Creation of Events Team</h2>
+          <p class="color-contrast-medium">
+            We form the Events Team to organise and run future events. Led by Kat and comprised by
+            staff members, the goal of the team is to ultimately host more events in a more
+            sustainable way.
+          </p>
```

Comment:
This may be sacrilege, but I wouldn't be opposed if the events team isn't timeline worthy. (Kat don't kill me. I'll send you :cookie:s)